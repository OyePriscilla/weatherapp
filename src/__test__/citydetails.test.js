import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Cities from '../components/cities';
import store from '../redux/configureStore';

describe('Test for Citydetails', () => {
  test('should render', () => {
    const city = {
      id: 1,
      humidity: '79',
      speed: '2.35',
      name: 'Aba',
    };
    const tree = renderer.create(
      <Provider store={store}>
        <Cities cities={city} key={1} />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
