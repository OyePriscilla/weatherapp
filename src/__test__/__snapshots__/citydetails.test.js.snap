// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Citydetails should render 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <input
    onChange={[Function]}
    placeholder="Search Cities in Nigeria"
    type="text"
  />
  <div
    style={
      Object {
        "width": "98%",
      }
    }
  />
</div>
`;
